### PR TITLE
[13.0][ADD] stock_move_line_partner_tree

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Adds new column "Partner" in stock.move.line tree

### CODE REVIEW

File: stock_move_line_partner_tree/views/stock_move_line_views.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_stock_move_line_tree_inherit">
+        <field name="name">stock.move.line (picking_partner_id)</field>
+        <field name="model">stock.move.line</field>
+        <field name="inherit_id" ref="stock.view_move_line_tree"></field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='location_id']" position="before">
+                <field name="picking_partner_id" />
```
Comment (dalonsod) [inline stock_move_line_partner_tree/views/stock_move_line_views.xml]: Improvement:
```suggestion
                <field name="picking_partner_id" optional="show"/>
```

File: stock_move_line_partner_tree/views/stock_move_line_views.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            </xpath>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="view_stock_move_line_search_inherit">
+        <field name="name">stock.move.line (picking_partner_id)</field>
+        <field name="model">stock.move.line</field>
+        <field name="inherit_id" ref="stock.stock_move_line_view_search"></field>
+        <field name="arch" type="xml">
```
Comment (dalonsod) [inline stock_move_line_partner_tree/views/stock_move_line_views.xml:18]: Could you add searching by partner?

**Review by dalonsod — CHANGES_REQUESTED** [review]
Locally tested, LGTM. Anyway, see comments

**Comment by ChristianSantamaria** [comment]
Done 👍

**Review by dalonsod — APPROVED** [review]

**Pull request merged**